### DISCUSSION TITLE
How to make missing key search to fail on runtime? (return null or throw Exception)

### DISCUSSION THREAD

**Discussion by bayraak** [vinissimus/next-translate#568] [Q&A]
Currently, if we pass a key that is not in any of the namespace it's returning the same key as a result from `t()` function.

How we can configure the `t('any-missing-key')` to throw an exception or return null or even empty string?

**Comment by aralroca** [comment]
@bayraak maybe is useful to you to use this function for the missing keys:

![image](https://user-images.githubusercontent.com/13313058/113699115-74666480-96d5-11eb-917a-c965679625f3.png)

https://github.com/vinissimus/next-translate#3-configuration

Currently is doing a `console.warn` as default, but you can throw an error or save it on a file all missing keys or whatever
